### PR TITLE
Text fields changed to hints

### PR DESCRIPTION
**Pull request by idiallo-su**
Small change
EditTexts' text fields changed to hints so that user does not have to clear before typing

### CODE REVIEW

**Comment by idiallo-su** [comment]
Second approver can delete this branch after it's merged

**Review by SamBatula — APPROVED** [review]
Ok

**Review by emosaku — APPROVED** [review]

**Pull request closed without merging**
